tests/src/main/java/org/hibernate/beanvalidation/tck/tests/validation/PropertyPathTest.java

```diff
@@ -309,9 +309,9 @@ public void testPropertyPathSet() {
 			@SpecAssertion(section = Sections.VALIDATIONAPI_CONSTRAINTVIOLATION, id = "r"),
 			@SpecAssertion(section = Sections.VALIDATIONAPI_CONSTRAINTVIOLATION, id = "s"),
 			@SpecAssertion(section = Sections.VALIDATIONAPI_CONSTRAINTVIOLATION, id = "t"),
-			@SpecAssertion(section = Sections.VALIDATIONAPI_CONSTRAINTVIOLATION, id = "ae"),
```

Comment:
No, it does not. I created another quick one to update the "a_" and did the "a_" -> "b_" manually.

Anyway, the wraparound is not done automatically. One day maybe but I didn't find it was time well spent as it's a rather narrow case and might require some effort.